Source/WTF/wtf/PlatformGTK.cmake

```diff
@@ -1,5 +1,9 @@
 set(WTF_OUTPUT_NAME WTFGTK)
 
+if (ENABLE_MALLOC_HEAP_BREAKDOWN)
+    add_subdirectory(glib)
+endif ()
```

Comment:
We don't use nested CMakeLists.txt below the "WebKit framework" level.

If this code used GLib, then I would say: "paste everything into PlatformWPE.cmake and PlatformGTK.cmake, accepting there will be some code duplication." But it doesn't use GLib or any other Linux platform libraries. Nothing stopping other ports like JSCOnly from using it, right?

So what I would do is put it all directly in Source/WTF/wtf/CMakeLists.txt and just guard it with `if (ENABLE_MALLOC_HEAP_BREAKDOWN AND NOT APPLE).

Reply:
This makes sense to me